qcodes/configuration/qcodesrc_schema.json

```diff
@@ -342,6 +342,15 @@
                     "type": "string",
                     "default": "qcodes_",
                     "description": "Prefix to add to filename for exporting datasets to disk after a measurement finishes. Only activated if export_type is set"
+                },
+                "export_name_elements": {
+                    "description": "elements to use in export name. Names will be formated like '{export_prefix}_{export_name_elements[0]}_{...}.file_extension}",
```

Comment:
what about allowing folks to provide a format string that use the "elements"? the default would be ``{prefix}_{captured_run_id}_{guid}`` (or so) and the format string give full flexibility for users and reliefs us from maintaining the conversion from "elements" to "post_fix" :) what do you think @jenshnielsen ?

Reply:
I am not really sure. It seems like one more thing that users have to get right and if they don't exporting will throw an exception. I would thing that being able to control the order of the elements should be enough

Reply:
ok, thats fair. then let's go with the elements for now :)